docs/data-structures/stack-data-structure.mdx

```diff
@@ -0,0 +1,467 @@
+---
+title: Stack
+slug: /data-structures/stack-data-structure
+---
+
+## Introduction
+
+Stack is an abstract data type (ADT) with a predefined capacity.
+It is an ordered collection of items into which new items may be inserted and from which items may be deleted from one end, called the top of the stack.
+It is a linear data structure that follows the ```LIFO``` (Last In First Out) or the ```FILO``` (First In Last Out) order.
+
+A pile of books or deck of cards can be thought of as an example of a stack in the real world.
+
+## Basic Operations
+
+- ```push()``` : Adds an element onto the stack
+- ```pop()``` : Removes an element from the stack
+- ```peek()``` : Returns the top element of the stack
+- ```is_empty()``` : Checks if a stack is empty or not
+- ```display()``` : Prints all the stack elements
```

Comment:
Just give the names: 
- Push
- Pop
- Peek
- Empty
- Display 

No need to show them as functions